README.md

```diff
@@ -22,13 +24,16 @@ docker run \
     --mount "type=bind,src=YOUR_OUTPUT,dst=/output_dir" \
     ome-tiff-tiler
 ```
+
 (Or `gehlenborglab/ome-tiff-tiler:latest` to pull from DockerHub.)
 
-Two environment variables should be set:
-- `CHANNEL_PAGE_PAIRS`: Space-delimited pairs of layer name and corresponding TIFF index.
-- `PREFIX`: Prefix to prepend to tile filenames.
+Four environment variables should be set:
```

Comment:
```suggestion
Environment variables should be set:
```

CHANNEL_PAGE_PAIRS and SERVER_URL are mutually exclusive, so you wouldn't have both, right?